app/controllers/collection_numbers_controller.rb

```diff
@@ -17,23 +17,23 @@ class CollectionNumbersController < ApplicationController
   ]
 
   # Displays matrix of selected CollectionNumber's (based on current Query).
-  def index_collection_number # :norobots:
+  def index_collection_number
     query = find_or_create_query(:CollectionNumber, by: params[:by])
     show_selected_collection_numbers(query, id: params[:id].to_s,
                                             always_index: true)
   end
 
   # Show list of collection_numbers.
-  def index # :norobots:
+  def index
     store_location
     query = create_query(:CollectionNumber, :all)
     show_selected_collection_numbers(query)
   end
 
-  alias_method :list_collection_numbers, :index
+  alias list_collection_numbers index
```

Comment:
RuboCop offense. See https://docs.rubocop.org/rubocop/cops_style.html#stylealias
and
https://rubystyle.guide/#alias-method-lexically